addons/web_tip/static/src/js/tip.js

```diff
@@ -261,6 +253,18 @@
             return def;
         },
 
+        //if needed, scroll prior to displaying the tip
```

Comment:
This comment makes no sense here. Either leave it in its original place or remove it altogether.